docs/s3gw-with-k8s.md

```diff
@@ -0,0 +1,68 @@
+<!-- markdownlint-disable no-duplicate-heading -->
+# Installing K8s for s3gw
+
+The following document describes the prerequisite installations to run the
+S3 Gateway (s3gw), an S3 object storage service, on top of a vanilla K8s
```

Comment:
"vanilla K8s" is the colloquial name I use for the K8s you get when you install them from Google's repos using `kubeadm`:
https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/
Note that it's probably quite different than what you'd get e.g. at a hosted K8s provider or when using other installers like the `kubespray` Ansible playbooks.

Reply:
To be clear, open to other ways of phrasing it: I just reviewed other terms and I liked this.

Reply:
It'll be fine. I'm just saying that it's quite hand-wavy and not very well defined.